client/app/components/cards-list/CardsList.jsx

```diff
@@ -33,34 +28,38 @@ export default class CardsList extends React.Component {
     searchText: '',
   };
 
+  constructor(props) {
+    super(props);
+    this.items = [];
+
+    let itemId = 1;
+    props.items.forEach((item) => {
+      this.items.push({ id: itemId, ...item });
+      itemId += 1;
+    });
+  }
+
   // eslint-disable-next-line class-methods-use-this
   renderListItem(item) {
-    const card = (
-      <Card
-        size="small"
-        cover={(<div><img alt={item.title} src={item.imgSrc} /></div>)}
-        onClick={item.onClick}
-        hoverable
-      >
-        <Meta title={(<h3>{item.title}</h3>)} />
-      </Card>
-    );
     return (
-      <List.Item className="cards-list-item">
-        {item.href ? (<a href={item.href}>{card}</a>) : card}
-      </List.Item>
+      <a key={`card${item.id}`} className="d-flex" href={item.href}>
+        <div className="visual-card" onClick={item.onClick}>
```

Comment:
Perhaps this would fix the clickable margin bug:

```html
<a key={`card${item.id}`} className="visual-card" onClick={item.onClick} href={item.href}>
  <img />
  <h3 />
</a>
```

Reply:
This one crossed my mind, but I didn't try it ^^. It worked, thanks @ranbena :slightly_smiling_face: